ui/components/app/custom-spending-cap/custom-spending-cap.js

```diff
@@ -6,15 +6,14 @@ import BigNumber from 'bignumber.js';
 import { I18nContext } from '../../../contexts/i18n';
 import Box from '../../ui/box';
 import FormField from '../../ui/form-field';
-import Typography from '../../ui/typography';
+import { Text } from '../../component-library';
 import { ButtonLink, Icon, ICON_NAMES } from '../../component-library';
```

Comment:
I just merged a PR that migrates the `Icon` component to TS. This may give you a conflict here. The correct imports should be:

```
import { Text, ButtonLink } from '../../component-library';
import { Icon, ICON_NAMES } from '../../component-library/icon/deprecated';
```

Apologies!